project-confero/confero
# Features/import data

# Problem

We want to be able to import data from the FEC into our own database

# Solution

- Add a custom manage command to parse the bulk CSVs and save them as model objects
- Also add some new templates and search functionality to make it easier to view the data.

## Change summary

- Set up importing
- Add committee and contribution models, remove Politician (since the FEC data don't have that concept)
- Add pagination
- Add search
- Make it pretty

## Steps to Verify

1. Follow the README to load the sample data
1. Start the server and see the data show up
1. Try searching for `C00648956` - that'll bring up a campaign with some contributions in the sample data

## Screenshot (Optional)

![index](https://user-images.githubusercontent.com/5178425/53336041-fb6b7080-38b1-11e9-8d4e-405c63f8013e.jpg)
![pelosi](https://user-images.githubusercontent.com/5178425/53336042-fb6b7080-38b1-11e9-8899-32b62fb36fe3.jpg)
![contributions](https://user-images.githubusercontent.com/5178425/53336040-fad2da00-38b1-11e9-9c7d-fb242e75a866.jpg)


# For Discussion

Changing the .env file to point to the RDS DB should let you load data into the remote DB
